spec/support/shared_examples/unprocessable_entity_response.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.shared_examples 'unprocessable entity response' do
+  it 'responds with 422 status' do
+    expect(response).to have_http_status(:unprocessable_entity)
+  end
+end
```

Comment:
```suggestion
shared_examples 'unprocessable entity response' do
  it 'responds with 422 status' do
    expect(response).to have_http_status(:unprocessable_entity)
  end
end
```

Generalmente además del status, este tipo de ejemplos tienen alguna forma de validar el mensaje de error. A fines del training esto es más que suficiente.